solutions/blocks/barchart/barchart.js

```diff
@@ -1,10 +1,8 @@
-import { decorateIcons } from '../../scripts/lib-franklin.js';
-
-export default async function decorate(block) {
+export default function decorate(block) {
   const ul = document.createElement('ul');
   [...block.children].forEach((row) => {
     const li = document.createElement('li');
-    li.innerHTML = row.innerHTML;
+    Array.from(row.children).forEach((child) => li.appendChild(child));
```

Comment:
this is great, this solves the issue; so in conclusion we only need to call the decorateIcons() function just 1 time, right ? also we have those remaining places that is also reproducing there: 
![image](https://github.com/hlxsites/bitdefender/assets/140399829/55278ed6-a25a-4a6c-9cbb-2cd2b8342b04)